src/Raven.Server/ServerWide/Commands/Sharding/UpdateValueForShardCommand.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+using Raven.Client.Extensions;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.ServerWide.Commands.Sharding;
+
+public abstract class UpdateValueForShardCommand : UpdateValueForDatabaseCommand
+{
+    protected UpdateValueForShardCommand() { }
+    protected UpdateValueForShardCommand(string databaseName, string uniqueRequestId) : base(databaseName, uniqueRequestId)
+    {
+    }
+
+    public string ShardName;
+    public Dictionary<string, string> CurrentChangeVectorPerShard;
```

Comment:
those do not look like related to generic command like 'UpdateValueForShardCommand'